src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/BrowserWebSocket.cs

```diff
@@ -467,11 +467,18 @@ public override Task CloseAsync(WebSocketCloseStatus closeStatus, string? status
 
         private Task CloseAsyncCore(WebSocketCloseStatus closeStatus, string? statusDescription, CancellationToken cancellationToken)
```

Comment:
This no longer needs to be a separate method, since it's no longer awaiting anything.  At that point you can then just combine this try/catch with the one in the caller.

Reply:
True but did not want to do that because the CloseAsyncCore is called from other places.

Reply:
Ok